### PR TITLE
Working on site

### PR DESCRIPTION
**Pull request by Keats**
And some tweaks as I write the docs

### CODE REVIEW

File: docs/content/documentation/content/linking.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Anchor insertion
+It is possible to have Gutenberg automatically insert anchor links next to the header, as you can see on the site you are currently 
+reading if you hover a title.
+
+This option is set at the section level, look up the `insert_anchor_links` variable on the 
+[Section front-matter page](./documentation/content/section.md#front-matter).
+
+The default template is very basic and will need CSS tweaks in your project to look decent. 
+If you want to change the anchor template, itt can easily be overwritten by
```
Comment (trishume) [inline docs/content/documentation/content/linking.md]: Typo `itt` -> `it`

File: components/front_matter/src/section.rs [hunk 2]
```diff
@@ -33,10 +33,10 @@ pub struct SectionFrontMatter {
     /// Path to be used by pagination: the page number will be appended after it. Defaults to `page`.
     #[serde(skip_serializing)]
     pub paginate_path: Option<String>,
-    /// Whether to insert a link for each header like in Github READMEs. Defaults to false
-    /// The default template can be overridden by creating a `anchor-link.html` template and CSS will need to be
-    /// written if you turn that on.
-    pub insert_anchor: Option<InsertAnchor>,
+    /// Whether to insert a link for each header like the ones you can see in this site if you hover one
+    /// The default template can be overridden by creating a `anchor-link.html` in the `templates` directory
+    /// This can also be set in a section front-matter if you only want it for
```
Comment (trishume) [inline components/front_matter/src/section.rs]: Sentence cut off: "if you only want it for"...

File: docs/content/documentation/content/section.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+# No pagination will happen if this isn't set or if the value is 0
+paginate_by = 0
+
+# If set, will be the path used by paginated page and the page number will be appended after it. 
+# For example the default would be page/1
+paginate_by = "page"
+
+# Whether to insert a link for each header like the ones you can see in this site if you hover one
+# The default template can be overridden by creating a `anchor-link.html` in the `templates` directory
+# This can also be set in a section front-matter if you only want it for
```
Comment (trishume) [inline docs/content/documentation/content/section.md]: "if you only want it for" also cut off afterwards here

**Review by trishume — COMMENTED** [review]
This looks awesome. I found some typos while reading.

**Comment by Keats** [comment]
Thanks a lot @trishume!

**Pull request merged**